backend/tests/routes/account.rs

```diff
@@ -81,3 +81,50 @@ async fn account_update() {
     let response = test::call_service(&app, request).await;
     assert!(response.status().is_success());
 }
+
+#[actix_web::test]
+async fn submit_personal_info() {
+    let test_database = TestDatabase::new().await;
+    let user = test_database.with_default_user().await;
+
+    let app = TestApp::default()
+        .with_database(test_database)
+        .build_app()
+        .await;
+
+    let request = test::TestRequest::post()
+        .uri("/account/submit_personal_information")
+        .insert_header(TestAuthHeader::new(user.clone()))
+        .set_json(serde_json::json!({
+          "birth_year": 1750,
+          "first_name": "Antonio",
+          "is_vegetarian": true,
+          "location": "Włochy",
+          "marketing_consent": true,
+          "organization": "Hack4Krak",
+          "referral_source": ["Linkedin"]
```

Comment:
We are not verifying birth year, and referral source? For birtheyar add like current year - 120 and maximum this year